storage/redis.go

```diff
@@ -332,8 +332,8 @@ func (r *RedisClusterStorageManager) GetAndDeleteSet(keyName string) []interface
 
 	var lrange *redis.StringSliceCmd
 	_, err := r.db.TxPipelined(func(pipe redis.Pipeliner) error {
-		lrange = pipe.LRange(fixedKey, 0, -1)
-		pipe.Del(fixedKey)
+		lrange = pipe.LRange(fixedKey, 0, chunkSize-1)
+		pipe.LTrim(fixedKey, chunkSize, -1)
```

Comment:
Are you sure that in case, if the chunk is not set, `LTrim(fixedKey, -1, -1)` is equal to `Del` call?

Reply:
You are right - it misses the last one... will re-work it a bit.

Reply:
@buger PTAL